source/documentation/gluster-hyperconverged/chap-Maintenance_and_Upgrading_Resources.html.md

```diff
@@ -29,5 +29,60 @@ NOTE: If geo-replication is setup on the gluster volumes, the geo-replication ne
     * Replace bricks from the host to be removed using the `Replace Brick` option
     * Once all bricks are replaces, the host can be moved to maintenance and removed.
 
+## Graceful shutdown and startup
+Currently there is no scirpt or single command available to shutdown a whole cluster. However, the following steps can be executed to manually shutdown and start a cluster again.
+
+### Shutdown
+1. Shutdown all VMs
+2. Enable global maintenance mode:
+    * In the Administration Portal, right-click the engine virtual machine, and select **Enable Global HA Maintenance**.
+    * You can also set the maintenance mode from the command line:
+    
+            # hosted-engine --set-maintenance --mode=global
+
+3. Wait for all VMs to be down
+4. If the cluster is running a hosted engine:
+   1. Logon to the node running hosted-engine
+   2. Shutdown hosted-engine
+   
+            # hosted-engine --vm-shutdown
+   
+   3. Wait for stopped hosted engine using
+   
+            # hosted-engine --vm-status
+
+5. Shutdown all nodes
+
+### Startup
+1. Switch on all nodes
+2. Start glusterd on all nodes since it does not start by default
+
+         # systemctl start glusterd
+
+3. Check peer and volume status on one of the nodes
+
+         # gluster peer status
+         # gluster volume status all
+   
+4. Wait for ovrt-ha-agent until 
+
+         # hosted-engine --vm-status
+   
+   does not fail anymore printing `The hosted engine configuration has not been retrieved from shared storage. Please ensure that ovirt-ha-agent is running and the storage server is reachable.`
+5. Start hosted engine on one of the nodes
+
```

Comment:
The HE VM will be started automatically - I think in Step 4. waiting for the HE health to be reported healthy would be sufficient.